### PR TITLE
[zh] translate docs/reference/using-api/health-checks.md

### PR DESCRIPTION
**Pull request by howieyuen**
fix https://github.com/kubernetes/website/issues/24500

### CODE REVIEW

File: content/zh/docs/reference/using-api/health-checks.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: Kubernetes API 健康端点
+reviewers:
+- logicalhan
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:4]: Please remove `reviewers` in localized version. They rarely speak Chinese.

File: content/zh/docs/reference/using-api/health-checks.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+The Kubernetes API server provides 3 API endpoints (`healthz`, `livez` and `readyz`) to indicate the current status of the API server.
+The `healthz` endpoint is deprecated (since Kubernetes v1.16), and you should use the more specific `livez` and `readyz` endpoints instead.
+The `livez` endpoint can be used with the `--livez-grace-period` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) to specify the startup duration.
+For a graceful shutdown you can specify the `--shutdown-delay-duration` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) with the `/readyz` endpoint.
+Machines that check the `health`/`livez`/`readyz` of the API server should rely on the HTTP status code.
+A status code `200` indicates the API server is `healthy`/`live`/`ready`, depending of the called endpoint.
+The more verbose options shown below are intended to be used by human operators to debug their cluster or specially the state of the API server.
+-->
+Kubernetes API 服务器提供 3 个 API 端点（`healthz`、`livez` 和 `readyz`）来表明 API 服务器的当前状态。
+`healthz` 端点已被弃用（自 Kubernetes v1.16 起），而应该使用更具体的 `livez` 和 `readyz` 端点。
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:39]:
```suggestion
`healthz` 端点已被弃用（自 Kubernetes v1.16 起），你应该使用更为明确的 `livez` 和 `readyz` 端点。
```

File: content/zh/docs/reference/using-api/health-checks.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+`livez` 端点可与 `--livez-grace-period` [标志](/docs/reference/command-line-tools-reference/kube-apiserver)一起使用，来指定启动持续时间。
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:40]:
```suggestion
`livez` 端点可与 `--livez-grace-period` [标志](/zh/docs/reference/command-line-tools-reference/kube-apiserver)一起使用，来指定启动持续时间。
```

File: content/zh/docs/reference/using-api/health-checks.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+为了正常关机，你可以使用 `/readyz` 端点并指定 `--shutdown-delay-duration` [标志](/docs/reference/command-line-tools-reference/kube-apiserver)。
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:41]:
```suggestion
为了正常关机，你可以使用 `/readyz` 端点并指定 `--shutdown-delay-duration` [标志](/zh/docs/reference/command-line-tools-reference/kube-apiserver)。
```

File: content/zh/docs/reference/using-api/health-checks.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+[+]autoregister-completion ok
+[+]poststarthook/apiservice-openapi-controller ok
+healthz check passed
+```
+
+<!-- 
+The Kubernetes API server also supports to exclude specific checks.
+The query parameters can also be combined like in this example:
+-->
+Kubernetes API 服务器也支持排除特性的检查项。
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:96]:
```suggestion
Kubernetes API 服务器也支持排除特定的检查项。
```

File: content/zh/docs/reference/using-api/health-checks.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+[+]poststarthook/apiservice-status-available-controller ok
+[+]poststarthook/kube-apiserver-autoregistration ok
+[+]autoregister-completion ok
+[+]poststarthook/apiservice-openapi-controller ok
+[+]shutdown ok
+healthz check passed
+```
+
+<!-- ## Individual health checks -->
+## 个人健康检查  {#individual-health-check}
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:130]:
```suggestion
## 独立健康检查  {#individual-health-check}
```

File: content/zh/docs/reference/using-api/health-checks.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+<!-- 
+Each individual health check exposes an http endpoint and could can be checked individually.
+The schema for the individual health checks is `/livez/<healthcheck-name>` where `livez` and `readyz` and be used to indicate if you want to check the liveness or the readiness of the API server.
+The `<healthcheck-name>` path can be discovered using the `verbose` flag from above and take the path between `[+]` and `ok`.
+These individual health checks should not be consumed by machines but can be helpful for a human operator to debug a system:
+-->
+每个单独的健康检查都会公开一个 http 端点，并且可以单独检查。
+单个运行状况检查的模式为 `/livez/<healthcheck-name>`，其中 `livez` 和 `readyz` 表明你要检查的是 API 服务器是否存活或就绪。
+`<healthcheck-name>` 的路径可以通过上面的 `verbose` 参数发现 ，并采用 `[+]` 和 `ok` 之间的路径。
+这些单独的健康检查不应由机器消耗，但对于操作人员调试系统而言，是有帮助的：
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:143]:
```suggestion
这些单独的健康检查不应由机器使用，但对于操作人员调试系统而言，是有帮助的：
```

File: content/zh/docs/reference/using-api/health-checks.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/using-api/health-checks.md:4]: copy that

File: content/zh/docs/reference/using-api/health-checks.md (hunk 5 above)
Comment (howieyuen) [reply content/zh/docs/reference/using-api/health-checks.md:96]: ops...typo

File: content/zh/docs/reference/using-api/health-checks.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/docs/reference/using-api/health-checks.md]: omg, forget it

File: content/zh/docs/reference/using-api/health-checks.md (hunk 6 above)
Comment (howieyuen) [reply content/zh/docs/reference/using-api/health-checks.md]: fixed

**Pull request merged**